app/views/partners/profiles/show/_agency_stability.html.erb

```diff
@@ -50,7 +50,7 @@
       <dd><%= humanize_boolean(partner.turn_away_child_care) %></dd>
 
       <address>
-        <strong>Program Address</strong> <br>
+        <strong>Program Address (If Different)</strong> <br>
```

Comment:
I'm questioning this one. Does it make sense to append the `(If different)` text if it's a `show`  action and it is displaying all the addresses?

Reply:
I think it's appropriate, assuming that the main address is not getting copied into the program address and shown.